app/src/main/java/com/feedhenry/securenativeandroidtemplate/navigation/Navigator.java

```diff
@@ -37,12 +42,17 @@ public void navigateToHomeView(BaseActivity activity) {
 
     public void navigateToAuthenticationView(BaseActivity activity) {
         AuthenticationFragment authFragment = new AuthenticationFragment();
-        loadFragment(activity, authFragment, AuthenticationFragment.TAG);
+        AuthHelper authHelper = new AuthHelper(context);
```

Comment:
it might be better to just create a single object for `AuthHelper`. This way it will be very easy to check if the user is currently authenticated without creating a new instance of this object every time